helpers/award_manipulator.py

```diff
@@ -13,6 +15,20 @@ def getCacheKeysAndControllers(cls, affected_refs):
         return CacheClearer.get_award_cache_keys_and_controllers(affected_refs)
 
     @classmethod
+    def postUpdateHook(cls, awards):
```

Comment:
This probably needs `updated_attr_list` as an additional arg, even if it's unused.